e2e/cypress/integration/enterprise/ldap/ldap_group_sync_spec.js

```diff
@@ -0,0 +1,343 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. #. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @ldap
+
```

Comment:
Please use preset timeout values on timeouts/waits

```suggestion

import * as TIMEOUTS from '../../../fixtures/timeouts';

```

Reply:
Done!